### DISCUSSION TITLE
Code Review Thread

### DISCUSSION THREAD

**Discussion by jwildfire** [SafetyGraphics/volcanoPlot#39] [General]
Post thoughts and comments about existing code here! Also look for other packages related to volcano plots and or AEs, and review the [safetyGraphics Chart Creation Vignette](https://github.com/SafetyGraphics/safetyGraphics/wiki/ChartConfiguration) and [cookbook](https://github.com/SafetyGraphics/safetyGraphics/wiki/Cookbook)

@bzkrouse @nandriychuk

**Comment by bzkrouse** [comment]
@jwildfire @nandriychuk - The example should be working in the dev_new branch. You can load the example data from data/test_data.csv into the app.
